master-firmware/src/strategy.cpp

```diff
@@ -516,7 +517,7 @@ struct BuildTowerLevel : actions::BuildTowerLevel {
         NOTICE("Building a tower level %d at %d %d", level, x_mm, y_mm);
 
         if (level == 0) {
-            if (!strategy_goto_avoid(MIRROR_X(m_color, x_mm), y_mm, MIRROR_A(m_color, -225), TRAJ_FLAGS_ALL)) {
+            if (!strategy_goto_avoid(MIRROR_X(m_color, x_mm), y_mm, MIRROR_A(m_color, -215), TRAJ_FLAGS_ALL)) {
```

Comment:
Tested for both colors ?

Reply:
yes, but still some issues with picking up the 5 cubes, I need to calibrate the odometry again, and check that the marks on the vynil are accurate :sweat_smile: